api/genomics/docs/wgs_cohort_extraction_setup.md

```diff
@@ -0,0 +1,47 @@
+## Environment Setup for WGS cohort extraction
+
+1. Create wgs-cohort-extraction service account.   
+    - This should match the value in WorkbenchConfig.wgsCohortExtraction.serviceAccount
+    - This will have to go through sysadmins for non test environments
+
+2. Register service account as a Terra user 
+     1. Grab sa-creds.json for the service account. I used gcloud admin UI.
+     2. ```
+        curl -H "$(oauth2l header --json sa-creds.json userinfo.email userinfo.profile cloud-billing)" \
+        -H "Content-Type: application/json" \
+        -X POST -d' ' \
+        https://sam.dsde-dev.broadinstitute.org/register/user/v1
+
+3. Create extraction billing project and workspace
+    - `./project.rb create-wgs-cohort-extraction-bp-workspace --project all-of-us-workbench-test --billing-project-name aou-terra-ops-test-10 --workspace-name aou-terra-ops-test-workspace-10 --owners eric.song@pmi-ops.org,songe@broadinstitute.org`
+    - Make note of the proxy group and pet service account that the script will print out at the end
+
+4. Grant pet service account some permissions through Terraform
+    - `bigquery.readSessionUser` and `bigquery.jobUser` on all researcher projects
+    - writer access on all researcher project buckets
+    - add to VPC whitelist
+    - See https://github.com/broadinstitute/terraform-terra/pull/143/files as an example for the test environment
+
+5. Create Workflow and share with wgs-cohort-extraction service account
```

Comment:
We should definitely figure out how to script this part. Will be tedious across all environments, for all updates